lib/modules/platform/github/issue.ts

```diff
@@ -0,0 +1,28 @@
+import { z } from 'zod';
```

Comment:
maybe name it `schema/issue`?

Reply:
Will be less complicated once the final functionality is implemented